### DISCUSSION TITLE
whisper API giving me an error Property API_KEY doesn't exist

### DISCUSSION THREAD

**Discussion by jpreethi1522** [chengsokdara/use-whisper#17] [Q&A]
i tried using this module but its giving me an error when i provide the api key
please help me how to resolve it

import { useWhisper } from '@chengsokdara/use-whisper'

const App = () => {
  

  const {
    recording,
    speaking,
    transcribing,
    transcript,
    pauseRecording,
    startRecording,
    stopRecording,
  } = useWhisper({
    apiKey: kd-uvhishsdbvjhdbvuydsbcjdhbvujsdhbv,
  })

  return (
    <div>
      <p>Recording: {recording}</p>
      <p>Speaking: {speaking}</p>
      <p>Transcribing: {transcribing}</p>
      <p>Transcribed Text: {transcript.text}</p>
      <button onClick={() => startRecording()}>Start</button>
      <button onClick={() => pauseRecording()}>Pause</button>
      <button onClick={() => stopRecording()}>Stop</button>
    </div>
  )
}
export default App

im getting error like

ERROR  ReferenceError: Property 'kduvhishsdbvjhdbvuydsbcjdhbvujsdhbv' doesn't exist

This error is located at:
    in App (created by withDevTools(App))
    in withDevTools(App)
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent), js engine: hermes

**Comment by chengsokdara** [comment]
Hello, first never include your token in public like this.

I would advice you to read about .env, the token should be in ur .env.

**Reply by jpreethi1522** [reply]
the token i have sent is duplicate a random characters
